enterprise/neo4j-enterprise/pom.xml

```diff
@@ -90,6 +89,11 @@
       <artifactId>neo4j-security-enterprise</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.neo4j</groupId>
+      <artifactId>neo4j-bolt</artifactId>
+      <version>${project.version}</version>
```

Comment:
There is another neo4j-bolt dependency defined below with scope test that can be removed now since scope compile also implies test.